orderbook/openapi.yml

```diff
@@ -349,6 +349,7 @@ components:
             owner (i.e. the order signer).
           $ref: "#/components/schemas/Address"
           nullable: true
+          required: false
```

Comment:
Didn't we just learn that all parameters of an object are optional by default? We shouldn't have to make it explicitly not required....

Reply:
If this is the case then I slightly prefer these semantics rather than having a list of all required fields below.

Reply:
I think the error may be caused by the fact that `required` is meant to be at the end of the list and usually indented one less. At least according to the errors I am seeing in the build this is what I had on this failing build in a different PR

https://github.com/gnosis/gp-v2-services/runs/2235945020

Reply:
makes sense. Removed it...